### DISCUSSION TITLE
Any plans to use Bitnami Helm Charts for this Valkey operator?

### DISCUSSION THREAD

**Discussion by arpan57** [hyperspike/valkey-operator#21] [Q&A] [answered]
Hi @dmolik  - 

I noticed that the https://github.com/bitnami/charts/tree/main/bitnami/valkey-cluster/ charts are pretty neat. 
On one of my minikube clusters, I could spawn a valkey cluster successfully. 

I was wondering if it would be easier to write an operator around the [bitnami](https://github.com/bitnami/charts/tree/main/bitnami/valkey-cluster/ ) helm charts - could reduce efforts and re-use whats already been developed. 

Thanks,
Arpan

**Comment by dmolik** [comment, marked as answer]
the helm chart was heavily used in development, and the operator is quickly reaching feature parity.

**Reply by arpan57** [reply]
Thanks for the quick response. 
Looks like its already used. searched for bitnami in the source :-)
